### PR TITLE
vimPlugins.vim-git: init at 2020-07-13

### PR DESCRIPTION
**Pull request by ivankovnatsky**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ivankovnatsky** [comment]
```console
nixpkgs-review pr 108205

1 package added:
vimPlugins.vim-git (init at 2020-07-13)

28 packages updated:
spacevim vimPlugins.coc-nvim (2020-12-30 → 2021-01-02) vimPlugins.completion-nvim (2020-12-31 → 2021-01-01) vimPlugins.defx-git (2020-12-17 → 2021-01-01) vimPlugins.deol-nvim (2020-12-30 → 2021-01-02) vimPlugins.nvim-highlite (2020-12-30 → 2020-12-31) vimPlugins.nvim-lspconfig (2020-12-30 → 2021-01-01) vimPlugins.nvim-lsputils (2020-12-29 → 2020-12-31) vimPlugins.nvim-treesitter (2020-12-30 → 2021-01-02) vimplugin-nvim-treesitter-context vimPlugins.nvim-treesitter-textobjects (2020-12-27 → 2021-01-01) vimPlugins.Spacegray-vim (2020-12-23 → 2021-01-02) vimPlugins.telescope-nvim (2020-12-31 → 2021-01-01) vimPlugins.vim-airline (2020-12-30 → 2021-01-01) vimPlugins.airline (2020-12-30 → 2021-01-01) vimPlugins.vim-clap (2020-12-16 → 2021-01-01) vimPlugins.vim-floaterm (2020-12-18 → 2021-01-01) vimplugin-vim-hexokinase vimPlugins.vim-lsp (2020-12-31 → 2021-01-02) vimPlugins.vim-ocaml (2020-10-11 → 2021-01-01) vimPlugins.vim-polyglot (2020-12-30 → 2021-01-01) vimPlugins.polyglot (2020-12-30 → 2021-01-01) vimPlugins.vim-visual-multi (2020-12-31 → 2021-01-01) vimPlugins.vim-vsnip (2020-12-19 → 2020-12-31) vimPlugins.vimsence (2020-11-02 → 2021-01-01) vimPlugins.vista-vim (2020-12-27 → 2021-01-01) vimPlugins.yats-vim (2020-12-06 → 2021-01-01) vimPlugins.YouCompleteMe (2020-12-16 → 2020-12-31)

27 packages built:
spacevim vimPlugins.Spacegray-vim vimPlugins.YouCompleteMe vimPlugins.airline vimPlugins.coc-nvim vimPlugins.completion-nvim vimPlugins.defx-git vimPlugins.deol-nvim vimPlugins.nvim-highlite vimPlugins.nvim-lspconfig vimPlugins.nvim-lsputils vimPlugins.nvim-treesitter vimPlugins.nvim-treesitter-context vimPlugins.nvim-treesitter-textobjects vimPlugins.polyglot vimPlugins.telescope-nvim vimPlugins.vim-clap vimPlugins.vim-floaterm vimPlugins.vim-git vimPlugins.vim-hexokinase vimPlugins.vim-lsp vimPlugins.vim-ocaml vimPlugins.vim-visual-multi vimPlugins.vim-vsnip vimPlugins.vimsence vimPlugins.vista-vim vimPlugins.yats-vim
```

**Comment by hugolgst** [comment]
```
hash mismatch in fixed-output derivation '/nix/store/wr6pmkd1abh2ysc8hx33jyjniwcbd6s2-maple-vendor.tar.gz':
  wanted: sha256:0000000000000000000000000000000000000000000000000000
  got:    sha256:1738hvqzwr4h1bigsqffc6alkzvir8s6f7mr0xyp21qbf5qkxmq2
```

File: pkgs/misc/vim-plugins/overrides.nix [hunk 1]
```diff
@@ -644,7 +644,7 @@ self: super: {
           libiconv
         ];
 
-        cargoSha256 = "QUi3GyAsakAtDQkiVA7ez05s5CixqsVSp92svYmcWdQ=";
```
Comment (hugolgst) [inline pkgs/misc/vim-plugins/overrides.nix:647]:
```
hash mismatch in fixed-output derivation '/nix/store/wr6pmkd1abh2ysc8hx33jyjniwcbd6s2-maple-vendor.tar.gz':
  wanted: sha256:0000000000000000000000000000000000000000000000000000
  got:    sha256:1738hvqzwr4h1bigsqffc6alkzvir8s6f7mr0xyp21qbf5qkxmq2
```

**Review by hugolgst — CHANGES_REQUESTED** [review]
```
1 package failed to build:
vimPlugins.vim-clap
```

File: pkgs/misc/vim-plugins/overrides.nix (hunk 1 above)
Comment (ivankovnatsky) [reply pkgs/misc/vim-plugins/overrides.nix:647]: fixed.

**Review by hugolgst — APPROVED** [review]
Looks better now :)

```
1 package added:
vimPlugins.vim-git (init at 2020-07-13)

28 packages updated:
spacevim vimPlugins.coc-nvim (2020-12-30 → 2021-01-02) vimPlugins.completion-nvim (2020-12-31 → 2021-01-01) vimPlugins.defx-git (2020-12-17 → 2021-01-01) vimPlugins.deol-nvim (2020-12-30 → 2021-01-02) vimPlugins.nvim-highlite (2020-12-30 → 2020-12-31) vimPlugins.nvim-lspconfig (2020-12-30 → 2021-01-01) vimPlugins.nvim-lsputils (2020-12-29 → 2020-12-31) vimPlugins.nvim-treesitter (2020-12-30 → 2021-01-02) vimplugin-nvim-treesitter-context vimPlugins.nvim-treesitter-textobjects (2020-12-27 → 2021-01-01) vimPlugins.Spacegray-vim (2020-12-23 → 2021-01-02) vimPlugins.telescope-nvim (2020-12-31 → 2021-01-01) vimPlugins.vim-airline (2020-12-30 → 2021-01-01) vimPlugins.airline (2020-12-30 → 2021-01-01) vimPlugins.vim-clap (2020-12-16 → 2021-01-01) vimPlugins.vim-floaterm (2020-12-18 → 2021-01-01) vimplugin-vim-hexokinase vimPlugins.vim-lsp (2020-12-31 → 2021-01-02) vimPlugins.vim-ocaml (2020-10-11 → 2021-01-01) vimPlugins.vim-polyglot (2020-12-30 → 2021-01-01) vimPlugins.polyglot (2020-12-30 → 2021-01-01) vimPlugins.vim-visual-multi (2020-12-31 → 2021-01-01) vimPlugins.vim-vsnip (2020-12-19 → 2020-12-31) vimPlugins.vimsence (2020-11-02 → 2021-01-01) vimPlugins.vista-vim (2020-12-27 → 2021-01-01) vimPlugins.yats-vim (2020-12-06 → 2021-01-01) vimPlugins.YouCompleteMe (2020-12-16 → 2020-12-31)

27 packages built:
spacevim vimPlugins.Spacegray-vim vimPlugins.YouCompleteMe vimPlugins.airline vimPlugins.coc-nvim vimPlugins.completion-nvim vimPlugins.defx-git vimPlugins.deol-nvim vimPlugins.nvim-highlite vimPlugins.nvim-lspconfig vimPlugins.nvim-lsputils vimPlugins.nvim-treesitter vimPlugins.nvim-treesitter-context vimPlugins.nvim-treesitter-textobjects vimPlugins.polyglot vimPlugins.telescope-nvim vimPlugins.vim-clap vimPlugins.vim-floaterm vimPlugins.vim-git vimPlugins.vim-hexokinase vimPlugins.vim-lsp vimPlugins.vim-ocaml vimPlugins.vim-visual-multi vimPlugins.vim-vsnip vimPlugins.vimsence vimPlugins.vista-vim vimPlugins.yats-vim
```

**Review by hugolgst — APPROVED** [review]

**Comment by hugolgst** [comment]
@ivankovnatsky is the PR still a draft? If not please update the status

**Comment by ivankovnatsky** [comment]
@hugolgst sorry, marked as ready, thank you for your time reviewing this.

**Pull request merged**